src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -443,18 +425,10 @@ void DoorLockServer::SetCredentialCommandHandler(
 {
     emberAfDoorLockClusterPrintln("[SetCredential] Incoming command [endpointId=%d]", commandPath.mEndpointId);
 
-    auto fabricIdx = getFabricIndex(commandObj);
-    if (kUndefinedFabricIndex == fabricIdx)
+    auto operationalId = getOperationalId(commandObj);
+    if (operationalId == OperationalId::UndefinedOperationalId())
```

Comment:
the logic does not seem the same: previous code was rejecting any pair of (undefinedFabric, validNode), (validFabric, undefinedNode) and (undefinedFabric, undefinedNode).

The new code only checks for (undefinedFabric, undefinedNode). I expect operationalId to provide a helper method that does the "isValid" check or simlarely named that will check fabric and node ids independently.